ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/xpn/minimal-permission/byo-hosted-zone/private/provision/cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-hosted-zone-private-provision-chain.yaml

```diff
@@ -0,0 +1,44 @@
+chain:
+  as: cucushift-installer-rehearse-gcp-ipi-xpn-minimal-permission-byo-hosted-zone-private-provision
+  steps: 
+    - ref: upi-conf-gcp-xpn
+    - ref: ignition-bastionhost
+    - ref: gcp-provision-bastionhost
+    - ref: proxy-config-generate
+    - chain: ipi-conf-gcp
+    - ref: ipi-conf-credentials-mode
+    - ref: ipi-conf-gcp-xpn-network
+    - ref: ipi-conf-gcp-create-firewall-rules
+    - ref: ipi-conf-gcp-basedomain
+    - ref: gcp-provision-private-dns-zone
+    - ref: gcp-provision-xpn-byo-hosted-zone-min-perm-passthrough-serviceaccount
+    - ref: ipi-conf-gcp-service-account
+    - ref: ipi-conf-gcp-private-dns-zone
+    - chain: ipi-install
+    - ref: enable-qe-catalogsource
+    - chain: cucushift-installer-check
+    - ref: cucushift-installer-check-gcp-private
+    - ref: cucushift-installer-check-gcp-credentials-mode
+  env:
+  - name: PUBLISH
+    default:  "Internal"
+  - name: CREDENTIALS_MODE
+    default: "Passthrough"
+    documentation: CCO mode.
+  - name: COMPUTE_NODES_TAGS
+    default: "preserved-ipi-xpn-compute"
```

Comment:
@jianli-wei could you help me understanding why we need such specific Tags for compute and master, is this required for this type of cluster deployment or is it just a cross-functional integration test?